package.json

```diff
@@ -93,7 +95,6 @@
     "nunjucks": "^3.2.4",
     "pino": "^9.0.0",
     "pug": "^3.0.2",
-    "simple-get": "^4.0.1",
     "split2": "^4.2.0",
     "tap": "^21.0.0",
```

Comment:
why is tap not being removed from dev dependencies?

Reply:
On node 20 snapshots are not available yet, so I cannot remove it for those tests yet

Reply:
https://github.com/fastify/point-of-view/pull/461#discussion_r1986731168